07_git_exercises/danielmalka_ex1/README.md

```diff
@@ -0,0 +1,49 @@
+# Git Basics:
+1. echo "1" > abc.txt
+2. red 
+3. after git add command on abc.txt - the color changed to green 
+4. echo "2" >> abc.txt 
+5. the color changed to red again (using git status command)
```

Comment:
The color should be blue (the question refer to PyCharm's project view, not to git)